src/App.test.js

```diff
@@ -1,9 +1,26 @@
-import React from 'react';
-import { render } from '@testing-library/react';
 import App from './App';
+import HomePage from './HomePage/HomePage';
+import Skills from './Skills/Skills';
+import React from 'react';
+import { mount } from 'enzyme';
+import { MemoryRouter } from 'react-router';
+
+it('/ route goes to home page', () => {
```

Comment:
Nitpick:

Reading up it looks like `it` is an alias of `test`, useful when we want to approximate english syntax when describing the behaviour of a particular entity/method, is that right? In this case I wonder if `test` would be more readable, as "it / route goes to home page" pulled me up short a bit.

Reply:
Excellent point! I'm used to using `it` in my tests so didn't give it a second thought, but will change to `test`